README.md

```diff
@@ -49,7 +49,7 @@ I need a full-stack web app that stores data in a Cosmos database.
 Web Template Studio approaches web app creation using the following four attribute sets:
 
 - **Project type**: First, what type of application are you building? We currently support only one type: _Fullstack Application_.
```

Comment:
Should we also update this since we removed Project Type page?

Reply:
good catch!